python/visualize/vid_overlay_position.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
```

Comment:
Is this file related to the python API changes in this PR or was this included by mistake?